blog/sudoko_blog_structure.md

```diff
@@ -247,4 +247,16 @@ How accurate? Compare to different implementations both AI and programmatic ones
 
 ## Conclusion
 
-Is using neural networks to solve sudoku a good idea? What did we learn from this task?
+We have learnt a great deal throughout this journey to solving sudoku puzzles with neural networks. We discovered how different architectures are suited to different tasks, how to apply improvement techniques to poorly performing base models, and how to work with datasets to train models for the task at hand.
+
+We have also learnt (confirmed?) that neural networks are not the best way to solve sudoku puzzles. There are numerous drawbacks including:
+
+- Poor accuracy, especially on puzzles with a great number of blanks
+- Solving different size puzzles requires re-training
```

Comment:
By 'different size puzzles,' I mean 4x4 or 12x12 sudokus instead of the standard 9x9. At the very least, we would have to change the number of inputs and outputs of the model, which in turn could cause the weights to lose any sense and potentially not run at all. Regardless, training our model does not take very long (comparatively) so this could be removed and instead made into a positive and aligned with the other comment concerning how we do not have to care about the inner workings of solving a sudoku (as we would in the case of coding a programmatic solver).

Reply:
Ah yes that's true, the architecture would have to be tweaked too so yes you're right - is that something which code solvers can handle typically? I think we can keep it in if so but just clarify what you mean by size maybe?

Reply:
Will do. I do not see why backtracking or brute force methods could not be subtly altered to support arbitrary sudoku sizes (just changing 9 to 12 in numbers to check for example). I am quite certain though that there will be optimisations made to many sudoku solvers which only hold for 9x9 standard sudokus. Will investigate to clarify.